completion/available/heroku.completion.bash

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
+if command -v heroku > /dev/null; then
+    heroku autocomplete > /dev/null
+    printf "$(heroku autocomplete:script bash)" >> ~/.bashrc; source ~/.bashrc
```

Comment:
There are several problems with this line:
* Doesn't this append the output of the `heroku` command to the `~/.bashrc` file every time you open a new terminal?
* Depending on your OS, `~/.bashrc` might not be the correct file, other OS's use `~/.bash_profile` for example.
* The purpose of Bash-it is that you don't need to append all of this code to your Bash init file, Bash-it should take care of loading the required content without having to edit your Bash init file.
* Sourcing `~/.bashrc` from Bash-it might lead to an endless loop, in case you're using `~/.bashrc` to load Bash-it itself.